src/quo2/components/buttons/button/component_spec.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo2.components.buttons.button.component-spec
```

Comment:
the only actual changes in this pr is that I used the test helpers for this test rather the old direct methods. However the tests are not changed at all 👍